sigle/src/pages/login.tsx

```diff
@@ -115,6 +133,113 @@ const Login = () => {
           </BlockIllustration>
         </Grid>
       </LoginContainer>
+      <Footer>
+        <Flex css={{ mb: '$5' }} gap="3">
+          <IconButton
+            as="a"
+            href={sigleConfig.twitterUrl}
+            target="_blank"
+            rel="noreferrer"
+          >
+            <TwitterLogoIcon />
+          </IconButton>
+          <IconButton
+            as="a"
+            href={sigleConfig.discordUrl}
+            target="_blank"
+            rel="noreferrer"
+          >
+            <DiscordLogoIcon />
+          </IconButton>
+          <IconButton
+            as="a"
+            href={sigleConfig.twitterUrl}
+            target="_blank"
+            rel="noreferrer"
+          >
+            <GitHubLogoIcon />
+          </IconButton>
+        </Flex>
+        <Flex
+          css={{
+            boxShadow: '0 1px 0 0 $colors$gray6',
+            pb: '$3',
+            justifyContent: 'center',
+            width: '100%',
+          }}
+          gap="5"
+        >
+          <Text
+            size="sm"
+            as="a"
+            target="_blank"
+            rel="noreferrer"
+            href={sigleConfig.blogUrl}
+            css={{
+              color: '$gray12',
+              '&:hover': {
+                boxShadow: '0 1px 0 0px $colors$gray12',
+              },
+            }}
+          >
+            Blog
+          </Text>
+          <Text
+            size="sm"
+            as="a"
+            target="_blank"
+            rel="noreferrer"
+            href={sigleConfig.documentationUrl}
+            css={{
+              color: '$gray12',
+              '&:hover': {
+                boxShadow: '0 1px 0 0px $colors$gray12',
+              },
+            }}
+          >
+            Documentation
+          </Text>
+          <Text
+            size="sm"
+            as="a"
+            target="_blank"
+            rel="noreferrer"
+            href={sigleConfig.discordUrl}
+            css={{
+              color: '$gray12',
+              '&:hover': {
+                boxShadow: '0 1px 0 0px $colors$gray12',
+              },
+            }}
+          >
+            Support
+          </Text>
+          <Text
+            size="sm"
+            as="a"
+            target="_blank"
+            rel="noreferrer"
+            href={sigleConfig.feedbackUrl}
+            css={{
+              color: '$gray12',
+              '&:hover': {
+                boxShadow: '0 1px 0 0px $colors$gray12',
+              },
+            }}
+          >
+            Feedback
+          </Text>
+        </Flex>
+        <Text
+          size="xs"
+          css={{
+            py: '$5',
+            color: '$gray12',
+          }}
+        >
+          © Sigle {new Date().getFullYear()} - SAS Incognito Lab
```

Comment:
can we remove `- SAS Incognito Lab` till we have the privacy policy ready please?